src/main/kubeconfig-manager.ts

```diff
@@ -41,7 +28,25 @@ export class KubeconfigManager {
     }
 
     return this.tempFile;
+  }
+
+  async unlink() {
+    if (!this.tempFile) {
+      return;
+    }
 
+    logger.info(`Deleting temporary kubeconfig: ${this.tempFile}`);
+    await fs.unlink(this.tempFile);
+    this.tempFile = undefined;
```

Comment:
Should be `null` to be consistent.

Reply:
I didn't change this code, only moved it's place.